docs/constitution.rst

```diff
@@ -0,0 +1,60 @@
+Mu's Constitution
+=================
+
+Our Values
+----------
+
+Happily, Mu's users and developers are a diverse and international community
+who maintain a reputation as a friendly, welcoming and dynamic group.
+
+We trust that such collaborators will treat each other in a way that reflects
+the widely help view that diversity and friendliness are strengths of our
+community to be celebrated and fostered.
+
+We believe anyone involved in Mu's community has a right to:
+
+* be treated with courtesy, dignity and respect;
+* be free from any form of discrimination, victimisation, harassment or
+  bullying.
+* enjoy an environment free from unwelcome behaviour, inappropriate language
+  and unsuitable imagery.
+
+To this end, we hope participants in our community abide by the spirit of our
+:doc:`code_of_conduct`.
+
+The Role of Administrators
+--------------------------
+
+Contributing to Mu
+------------------
+
+Managing Expectations
+---------------------
+
+In Case of Problems
+-------------------
+
+Problems within any community are inevitable. We hope, if at all possible, that
+folks try to work out their differences with compassion, patience and
+forgiveness. However, we understand that sometimes, problems cannot be resolved
+in such a way.
+
+In this case please contact the administrators via email: <email TBC>
+
+Any contact will remain confidential, be taken seriously, investigated, and
+dealt with sensitively. Mu admins reserve the right to carry out any of the
+following actions relating to community related problems:
```

Comment:
"relating to ... related" is a bit awkward. Perhaps "concerning community-related problems"